esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
+{% autoescape off %}
+{{ msgbdy }}
+{% endautoescape %}
```

Comment:
Yes, sorry. I mean for humans to handle the request if logging in is annoying or they forgot their password.